AR/Packages/packages-lock.json

```diff
@@ -164,15 +164,16 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.xr.interaction.toolkit": {
-      "version": "1.0.0-pre.6",
+      "version": "1.0.0-pre.8",
       "depth": 0,
       "source": "registry",
       "dependencies": {
         "com.unity.inputsystem": "1.0.2",
         "com.unity.ugui": "1.0.0",
+        "com.unity.xr.legacyinputhelpers": "2.1.7",
```

Comment:
XRIT depends on `2.1.8` now, should we update here?

Reply:
We only bumped that for `2.0`, `1.0.0-pre.8` is still dependent on `2.1.7`.